doc/grobi.conf

```diff
@@ -98,6 +98,24 @@ rules:
     execute_after:
       - xautolock -disable
 
+   # Setup external monitor and mirror laptop
+  - name: Mirror
+    outputs_connected: [eDP1, DP2]
+    outputs_present: [eDP1, DP2]
+    outputs_same_as:
+      - output: DP2
+        same_as: eDP1
```

Comment:
Hm, this rule looks odd. You are trying to configure two displays showing the same content, right? How about we add `configure_mirror` instead of using `configure_row` and `outputs_same_as`? This would simplify the rule a lot:

```yaml
outputs_present: [eDP1, DP2]
configure_mirror: [eDP1, DP2]
```

Reply:
I like the name configure_mirror :) but if I add them to a list (like outputs_present) it would indicate that you could specify more than 2 displays but that is not really possible when mirroring. I can do the change if you still prefer it to be like output_present

Reply:
Hm, are you sure that's the case for all graphic cards? I can imagine it's a common limitation (only two outputs in mirror mode), but I guess there are devices where three is also possible.

I'd prefer `configure_mirror`, you can then remove the code for `outputs_same_as`, which makes the configuration simpler :)

Reply:
I'll have a look at it later and do a fix :)

Reply:
Fixed and I rebased it the newly added features